app/views/create/fromimage.html

```diff
@@ -14,10 +14,7 @@
               <div class="col-md-12">
                 <div class="osc-form">
                   <div class="row">
-                    <div class="col-md-2 template-name gutter-top hidden-sm hidden-xs">
-                      <span class="fa fa-cubes"></span>
-                    </div>
-                    <div class="col-md-8">
```

Comment:
This might be the only one I keep. If we remove it from this page, we should remove it from the template page as well. But I'd rather use the technology icon for the builder / template.

@jwforres thoughts?